adguard/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BUILD_FROM=ghcr.io/hassio-addons/base:13.0.0
```

Comment:
This is setting a default, but doens't upgrade it all. Did you test this?

If so, how?

Reply:
Cool, but that is not what I asked :)

Could you maybe answer my question above? Thanks! 👍 

../Frenck

Reply:
? LOL?

It was a fairly simple and straightforward question. How did you test the changes?

Considering you are closing the PR, I assume it was a drive-by PR that wasn't tested?

../Frenck